ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -164,16 +164,6 @@
       priority: 50
   when: pulp_build == "beta" or pulp_build == "stable"
 
-- name: Check fips state
-  shell: cat /proc/sys/crypto/fips_enabled
-  register: fips_state
-
-- name: Install yum-plugin-priorities
-  package:
-    name: yum-plugin-priorities
-    state: present
-  when: fips_state.stdout == '1'
```

Comment:
Instead, you can probably drop this "when" conditional to ensure it's available across the board.

Reply:
There was another playbook doing the same. I can modify this, and delete the playbook.

Reply:
ah, then never-mind, feel free to do the other.